src/customStyles.css

```diff
@@ -1,3 +1,7 @@
 .leaflet-bottom .leaflet-control {
   opacity: 0.6 !important;
+}
+
+.css-yk16xz-control {
```

Comment:
Where does this show up?

Reply:
the dropdown

Reply:
![Screen Shot 2019-09-04 at 12 34 09 PM](https://user-images.githubusercontent.com/6288799/64285318-64dad480-cf10-11e9-9221-e01a248fc6d5.png)

Reply:
Where _in the code_ does this css class show up?